examples/boussinesq_2d_imex/playground.py

```diff
@@ -36,12 +36,12 @@
     swparams['do_LU'] = True
 
     sparams = {}
-    sparams['maxiter'] = 12
+    sparams['maxiter'] = 16
 
     # setup parameters "in time"
     t0     = 0
-    Tend   = 3000
-    Nsteps =  500
+    Tend   = 384
+    Nsteps =  128
```

Comment:
The GS example takes 640 steps.. can you extend this a little bit? We have randomized fault injections, so we need many steps to get good statistics.